lava/magma/core/model/c/ports.h

```diff
@@ -0,0 +1,26 @@
+#ifndef _PORTS_H
+#define _PORTS_H
+#include <stdint.h>
+
+#ifdef PYTHON
+#include <Python.h>
+typedef PyObject Port;
+#else // HOST or EMBEDDED
```

Comment:
HOST and EMBEDDED likely require different treatment. 

On EMBEDDED you will purely use Andrew's CSP channel implementation while on HOST you might use Andrew's CSP API and may have to communicate with the SH either somehow via GRPC or via Ray in the future.

Reply:
Yes. Haven't gotten there yet.